beacon-chain/p2p/handshake.go

```diff
@@ -86,12 +89,11 @@ func (s *Service) AddConnectionHandler(reqFunc func(ctx context.Context, id peer
 				validPeerConnection := func() {
 					s.peers.SetConnectionState(conn.RemotePeer(), peers.PeerConnected)
 					// Go through the handshake process.
-					multiAddr := fmt.Sprintf("%s/p2p/%s", conn.RemoteMultiaddr().String(), conn.RemotePeer().String())
 					log.WithFields(logrus.Fields{
 						"direction":   conn.Stat().Direction,
-						"multiAddr":   multiAddr,
+						"multiAddr":   peerMultiaddrString(conn),
 						"activePeers": len(s.peers.Active()),
-					}).Info("Peer Connected")
+					}).Info("Peer connected")
```

Comment:
change this back too

Reply:
Hi, I changed this too in response to https://github.com/prysmaticlabs/prysm/pull/6409#discussion_r446342716

Reply:
ok thanks that makes sense